enterprise/server/test/webdriver/invocation/invocation_test.go

```diff
@@ -78,3 +78,73 @@ func TestAuthenticatedInvocation_LogUploadEnabled(t *testing.T) {
 	// default BB org, and make sure we get PermissionDenied instead of the
 	// login page
 }
+
+func TestAuthenticatedInvocation_Dev_LogUploadEnabled(t *testing.T) {
```

Comment:
I know this is WIP so maybe you're already thinking of this -- but I was thinking that rather than having separate `_Dev` suffixes etc., we can abstract out any env-specific config/logic into an interface like `WebdriverTarget` or something (where the `Local` target maybe has an `Init()` implementation that actually spins up the server) and then enable those things explicitly via test flags / env vars